Makefile

```diff
@@ -88,7 +88,7 @@ init:  get-saxon  get-xspec get-rdflib
 ############################ Main tasks ##############################################
 # Run unit_tests
 unit-tests:
-	@ant -lib ${SAXON} unit_tests
+	@mvn install
```

Comment:
Perhaps it is more correct to use the otehr 2 commands with mvn dependecies download and dependecies copy. 

It is upt o you to decie.